djconnectwise/api.py

```diff
@@ -65,6 +65,11 @@ class ConnectWiseRecordNotFoundError(ConnectWiseAPIClientError):
     pass
 
 
+class ConnectWiseSecurityPermissionsException(ConnectWiseAPIClientError):
+    """The record was not found."""
```

Comment:
This comment should updated or removed. I'm guessing its a copy/paste from the class above.

Reply:
Ah, yes I meant to update that.